README.md

```diff
@@ -47,8 +47,8 @@ Fixing build problems with dependencies or cached build options
 ---------
 
 Redis has some dependencies which are included into the `deps` directory.
-`make` does not rebuild dependencies automatically, even if something in the
-source code of dependencies is changes.
+`make` does not rebuild dependencies automatically, even if something in
```

Comment:
For English grammar, since Salvatore brought it up:
"does not automatically rebuild dependencies, even if a dependency's source changes."

Is simple and correct. The comma is optional. ;)